tests/functional/reset_password.js

```diff
@@ -100,8 +102,8 @@ define([
       .then(openPage(url, header));
   });
 
-  function testAtSettingsWithVerifiedMessage(context) {
-    return context.remote
+  var testAtSettingsWithVerifiedMessage = thenify(function() {
```

Comment:
What are our plans for `var`, generally? If we have a plan to one day lint them out of existence maybe `const` this instead?